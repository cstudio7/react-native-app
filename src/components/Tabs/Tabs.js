import React from 'react';
import { View, StyleSheet } from 'react-native';
import Spacing from '../../constants/Spacing';

export default class Tabs extends React.Component {
  render() {
    return <View style={styles.tabs}>{this.props.children}</View>;
  }
}

const styles = StyleSheet.create({
  tabs: {
    justifyContent: 'space-between',
    flexDirection: 'row'
  }
});
