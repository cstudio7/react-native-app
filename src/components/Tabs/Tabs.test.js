import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';

import Tabs from './Tabs';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Tabs>
        <Text>Hello, world!</Text>
      </Tabs>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
