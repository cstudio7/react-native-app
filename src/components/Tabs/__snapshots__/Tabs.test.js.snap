// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "marginBottom": 15,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello, world!
  </Text>
</View>
`;
